code/__DEFINES/flags.dm

```diff
@@ -69,6 +69,7 @@
 #define NOBLOOD		1024
 #define NOFIRE		2048
 #define VIRUSIMMUNE	4096
+#define HARDFEET	8192

```

Comment:
We probably should use normal numbers instead of bitflag-like ones here since they're not used as bitflags. Not that it's related to change itself